instrumentation/jdbc/src/test/groovy/SqlNormalizerTest.groovy

```diff
@@ -139,19 +137,4 @@ class SqlNormalizerTest extends AgentSpecification {
       JDBCUtils.normalizeSql(sb.toString())
     }
   }
-
-  def "config can disable sql normalizer"() {
```

Comment:
Why removed?

Reply:
I had two options here:

1) store the property in a static final constant and remove this test (changing property value would require unloading & loading `JDBCUtils`);
2) leave this test and call `Config` on each `normalizeSql()` call.

I suspect that `normalizeSql()` can be called very often (each SQL query), so I've decided to choose the first option for performance reasons.